src/error-wrapper.ts

```diff
@@ -1,10 +1,10 @@
-import { FindComponentSelector } from './types'
+import { FindComponentSelector, DOMWrapperAPI } from './types'
 
 interface Options {
   selector: FindComponentSelector
 }
 
-export class ErrorWrapper {
+export class ErrorWrapper implements DOMWrapperAPI {
```

Comment:
nit: add some docs as well?

Reply:
When I finish refactoring and we are happy with the API :D